src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -63,13 +64,33 @@ const PromoListComponent = ({ promoItems, dir }) => {
     }
   };
 
+  const StoryPromoLiFeatures = styled(StoryPromoLi)`
+    line-height: 0;
+
+    &:first-child {
+      padding: 0 0 0.5rem 0;
+    }
+
+    &:not(:first-child):not(:last-child) {
+      padding: 0.5rem 0 0.5rem 0;
+    }
+
+    &:last-child {
+      padding: 0.5rem 0 0 0;
+    }
+  `;
+
```

Comment:
As part of the work for tablet layout, it may be worthwhile bringing the Psammead `StoryPromoUL` and `StoryPromoLi` components into Simorgh, as there are some styling that shouldn't be there in tablet layouts (namely the subtle grey divider). Not sure if this will help the duplication here mind!